src/wizard/tests/unit/Wizard.spec.tsx

```diff
@@ -0,0 +1,237 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+const { it, describe } = intern.getInterface('bdd');
+const { assert } = intern.getPlugin('chai');
+import testHarness from '@dojo/framework/testing/harness/harness';
```

Comment:
Do you think you could try using the new test renderer?
docs: https://next.dojo.io/learn/testing/introduction#using-assertions

`import renderer, { assertion } from '@dojo/framework/testing/renderer';`

Reply:
Ant recently re-wrote the `List` tests using the new test renderer, would be a good place to start looking: https://github.com/dojo/widgets/blob/master/src/list/tests/List.spec.tsx